reconcile/utils/terrascript_client.py

```diff
@@ -1742,7 +1742,7 @@ def populate_tf_resource_elasticache(self, resource, namespace_info,
         output_name_0_13 = output_prefix + '__db_endpoint'
         # https://docs.aws.amazon.com/AmazonElastiCache/
         # latest/red-ug/Endpoints.html
-        if values.get('cluster_mode'):
```

Comment:
the `cluster_mode` field is coming from the defaults file defined for the resources in app-interface fwiw